widget/assets/socWidgetInput.json

```diff
@@ -86,165 +86,186 @@
   "impactAnalysis": [
     {
       "id": "idArtifactsDivision",
-      "title": "Artifacts analysed",
-      "value": "?"
+      "idFromUser": "artifactsAnalysed",
+      "title": "Key not found",
```

Comment:
1. why title has changed to error value?
2. why do you need idFromUser field?
3. Please remove id from idArtifactsDivision as discussed
4. IMO, you can use const in js code for a common error message like "key not found", having it in json doesn't add any value

Reply:
idFromUser to map, required iD's with id's provided by user, I have changed the id's as discussed